libs/dh/market-participant/actors/feature-delegation/src/lib/create/dh-delegation-create-modal.component.ts

```diff
@@ -169,7 +171,18 @@ export class DhDelegationCreateModalComponent extends WattTypedModal<DhActorExte
   }
 
   private getDelegatedProcesses() {
-    return dhEnumToWattDropdownOptions(DelegatedProcess, this.getDelegatedProcessesToExclude());
+    return dhEnumToWattDropdownOptions(
+      DelegatedProcess,
+      this.getDelegatedProcessesToExclude()
+    ).filter((x) => {
+      if (this._featureFlagsService.isEnabled('process-delegation-allow-rsm12')) {
```

Comment:
Suggestion: Move this into a function with a descriptive name, it will make it easier to read and understand later :-)